lib-ee/emqx_ee_bridge/src/emqx_ee_bridge_mysql.erl

```diff
@@ -54,6 +54,19 @@ values(post) ->
             password => <<"">>,
             auto_reconnect => true
         },
+        resource_opts => #{
+            health_check_interval => ?HEALTHCHECK_INTERVAL_RAW,
+            start_after_created => ?START_AFTER_CREATED,
+            start_timeout => ?START_TIMEOUT_RAW,
+            auto_restart_interval => ?AUTO_RESTART_INTERVAL_RAW,
+            query_mode => sync,
+            async_inflight_window => ?DEFAULT_INFLIGHT,
+            enable_batch => false,
+            batch_size => ?DEFAULT_BATCH_SIZE,
+            batch_time => ?DEFAULT_BATCH_TIME,
+            enable_queue => false,
+            max_queue_bytes => ?DEFAULT_QUEUE_SIZE
+        },
```

Comment:
The resource_opts `start_after_created`, `start_timeout` should not be exported to the user, we should always hard code `start_after_create` = true, `start_timeout` to the default.

We also don't need export the `query_mode`, `async_inflight_window` opts to users, because our mysql driver doesn't support async query.